repos/os/include/net/ipv4.h

```diff
@@ -66,8 +66,8 @@ class Net::Ipv4_packet
 		 ** IPv4 header fields **
 		 ************************/
 
-		unsigned         _version         : 4;
 		unsigned         _header_length   : 4;
+		unsigned         _version         : 4;
```

Comment:
why do you change the order of version and IHL header fields here? The standard says that the version field is followed by the header length field, isn't it?